Here we fine tune slides for tomorrow's presentation (#22)
coderefinery/presentations: 2024_acm_hpcsig_edu.md

<img src="img/monitor-setup.jpg"
     alt="Streaming setup during Python for Scientific Computing"
     style="height: 200px;"/>

0.9 FTE (2 persons) + [10 persons in-kind + volunteers](https://coderefinery.org/about/contributors/)

<img src="img/CR_partners.png"
     alt="CodeRefinery partner logos"
     style="width: 750px;"/>

Co-advertize and co-organize with us

<img src="img/logos/tud.png"
     alt="TU Delft logo"
     style="height: 35px;"/>
<img src="img/logos/nl-escience-center.png"
     alt="The Netherlands eScience Center logo"
     style="height: 35px;"/>
<img src="img/logos/vu.png"
     alt="VU Amsterdam logo"
     style="height: 40px;"/>

---

# What we have learned

## About motivating/teaching

- Teaching isn't a lecture anymore. It's more
  .emph[like a live TV production], which *can* be as interactive as people in
  a room.

- .emph[Co-teaching] is a great way to onboard, get better quality, and reduce
  stress

- .emph[Good enough practices] better than perfect practices not
  applied

- Instead of "good for others": ".emph[good for your future you] and as side
  effect good for others"

---

# What we have learned

## About scaling

> - "bring your own classroom" seems to be a way to scale

Reviewer: ```suggestion
- .emph["bring your own classroom"] seems to be a way to scale
```

- .emph[Installation instructions and on-boarding] become more important

- We don't "see" classrooms -> .emph[feedback mechanism] in Q&A doc

- Make exercises longer to .emph[give classrooms the chance to interact]

---

# Collaborative document: Markdown

- Interactive, anonymous, parallel, async
- New question every 1-2 minutes!
- ASCII-graph feedback

<img src="img/notes.png"
     alt="Screenshot of exercise title and questions in collaborative notes"
     style="width: 100%;" />

We publish Q&A for each workshop: [Example](https://coderefinery.github.io/2024-03-12-workshop/questions/)

---

.left-column50[
## Future: Community project

> - Communicate value for volunteers and organizations
> 
> - Research groups send their students to us instead of
>   creating isolated material
> 
> - More collaboration with similar projects ("helper exchange program")
> 
> - Governance is community-driven

Reviewer: ```suggestion
- .emph[Communicate value] for volunteers and organizations

- Research groups send their students to us instead of
  creating isolated material

- .emph[More collaboration] with similar projects ("helper exchange program")

- Governance is .emph[community-driven]
```

> - Governance is community-driven

Reviewer: What is actually meant by this?

Author: What I meant is: community drafts the governance and a board governs the project with representatives from partners and community-elected member(s). I somehow wanted to write: not a top-down thing but we as the community define how the project should be governed in future which we are in the process to do.

]

.right-column50[
## Teaching format

> - Continue large-scale workshops
> 
> - Support local events
> 
> - More asynchronous content coupled with online events

Reviewer: ```suggestion
- Continue .emph[large-scale workshops]

- Support .emph[local events]

- More asynchronous content coupled with online events
```

> - More asynchronous content coupled with online events

Reviewer: Same as above, what is meant by this?

Author: Asynchronous content (let's find a better name): online videos and material that can be consumed asynchronously but then we also have events where we bring people together for demos/Q&A/hackathon.

Reviewer: Is this maybe known as "flipped classroom approach" in education world?

Author: Yes. I now hopefully clarified it.

]

---

## How you or your organization can participate

.center[
<img src="img/participate_organization_byoc.png"
     alt="Graphics that summarizes how organizations can participate: by advertizing, by sending observers or organizing local teams or through in-kind support"
     style="height: 400px;"/>
]

- **Join our next workshop** autumn 2024; follow our [newsletter](https://coderefinery.org/about/newsletter/) to get involved
- Tell your students and researchers about it
- Send one or more **exercise teams** or **join as observer**
- Use our material and give feedback

---

# We try to make it easier to join

.left-column60[
- **Chat with us**: https://coderefinery.zulipchat.com (ask questions about coding or learn about new tools)

> - [Blog](https://coderefinery.org/blog/),
>   [Newsletter and chat digest](https://coderefinery.org/about/newsletter/),
>   [Twitter](https://twitter.com/coderefine),
>   [Mastodon](https://fosstodon.org/@coderefinery),
>   [Support](support@coderefinery.org)

Reviewer: ```suggestion
- [Blog](https://coderefinery.org/blog/),
- [Newsletter and chat digest](https://coderefinery.org/about/newsletter/),
- [X/Twitter](https://twitter.com/coderefine),
- [Mastodon](https://fosstodon.org/@coderefinery),
- [Support e-mail](support@coderefinery.org)
```

]

.right-column40[
<img src="img/CR_teammeeting_mastodon_post.jpg"
     alt="Screenshot of a weekly team meeting summary posted on Mastodon"
     style="height: 450px;"/>
]

---

class: center, middle, inverse

# [Nordic RSE](https://nordic-rse.org/) Conference

May 30 - 31, at Aalto University campus in Otaniemi, Espoo, Finland

[https://nordic-rse.org/events/2024-in-person-conference/](https://nordic-rse.org/events/2024-in-person-conference/)

---
class: center, middle, inverse

## These slides: TODO

---

# Credits and license

## Text

- All text: CodeRefinery project, CC-BY 4.0

## Images

- Slide 3: H. Seibold, "6 helpful steps for reproducible research", CC-BY 4.0

> - Slides 6, 11, 12, 18: S. Wittke

Reviewer: ```suggestion
- Slides 7,16: S. Wittke
```

> - Slide 14: ATC tower, P. R. Miller, CC-BY 2.0
> - Slide 14: Monitor setup, R. Darst
> - Slide 14: Logos, (c) respective organizations

Reviewer: ```suggestion
- Slide 11: ATC tower, P. R. Miller, CC-BY 2.0
- Slide 11: Monitor setup, R. Darst
- Slide 11: Logos, (c) respective organizations
```

<img width="80px" src="img/deic.png" alt="logo: Danish e-Infrastructure Consortium">
<img width="90px" src="img/enccs.png" alt="logo: EuroCC National Competence Center Sweden (ENCCS)">
<img width="90px" src="img/naiss_black.png" alt="logo: National Academic Infrastructure for Super­computing in Sweden (NAISS)">
<img width="90px" src="img/nris.png" alt="logo: NRIS/Sigma2">
<img width="90px" src="img/sigma2.png" alt="logo: NRIS/Sigma2">

Co-advertize and co-organize with us

<img src="img/logos/tud.png"
     alt="TU Delft logo"
     style="height: 35px;"/>
<img src="img/logos/nl-escience-center.png"
     alt="The Netherlands eScience Center logo"
     style="height: 35px;"/>
<img src="img/logos/vu.png"
     alt="VU Amsterdam logo"
     style="height: 40px;"/>

---

# Connection to high-performance computing

- Many partners with HPC involvement

- Aalto University very active in inviting others to join HPC kickstart

- [Tuesday Tools & Techniques for High Performance Computing](https://scicomp.aalto.fi/training/scip/ttt4hpc-2024/)

- [ENCCS instructor training](https://enccs.github.io/instructor-training/) -> Materials based on CR train-the-trainer

> - NRIS (Norway) and LUMI (Finland) have adopted our training approach and material as template

Reviewer: ```suggestion
- NRIS (Norway) and LUMI (EuroHPC-JU) have adopted our training approach and material as template
```
